subgraphs/enzyme-core/entities/AaveV3DebtPosition.ts

```diff
@@ -0,0 +1,65 @@
+import { logCritical, uniqueEventId } from '@enzymefinance/subgraph-utils';
+import { Address, BigInt, ethereum } from '@graphprotocol/graph-ts';
+import {
+  AaveV3DebtPosition,
+  AaveV3DebtPositionChange,
+  Vault,
+  AssetAmount,
+  ExternalPositionType,
+} from '../generated/schema';
+import { getActivityCounter } from './Counter';
+import { useVault } from './Vault';
+
+export function useAaveV3DebtPosition(id: string): AaveV3DebtPosition {
+  let cdp = AaveV3DebtPosition.load(id);
+  if (cdp == null) {
+    logCritical('Failed to load fund {}.', [id]);
+  }
+
+  return cdp as AaveV3DebtPosition;
+}
+
+export function createAaveV3DebtPosition(
+  externalPositionAddress: Address,
+  vaultAddress: Address,
+  type: ExternalPositionType,
+): AaveV3DebtPosition {
+  let aaveV3DebtPosition = new AaveV3DebtPosition(externalPositionAddress.toHex());
+  aaveV3DebtPosition.vault = useVault(vaultAddress.toHex()).id;
+  aaveV3DebtPosition.active = true;
+  aaveV3DebtPosition.type = type.id;
+  aaveV3DebtPosition.eModeCategoryId = 0;
+  aaveV3DebtPosition.save();
+
+  return aaveV3DebtPosition;
+}
+
+// 255 is the max value of uint8, so we use 255 to indicate null
+// Unfortunately, it seems that we can't set eModeCategoryId to null
```

Comment:
You can set `eModeCategoryId` to null if you define it as optional in the schema.